scripts/install-ubuntu-22.04.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+# This script installs Klipper on an Ubuntu 22.04 ("Jammy") machine
+
+PYTHONDIR="${HOME}/klippy-env"
+SYSTEMDDIR="/etc/systemd/system"
+KLIPPER_USER=$USER
+KLIPPER_GROUP=$KLIPPER_USER
+
+# Step 1: Install system packages
+install_packages()
+{
+    # Packages for python cffi
+    PKGLIST="virtualenv python3-dev libffi-dev build-essential"
```

Comment:
**python3-dev** needs to be replaced with **python-dev-is-python3**. Errors out during installation. I replaced manually and it seems to fix this.

"
Package python-dev is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  python2-dev:i386 python2:i386 python2-dev python2 python-dev-is-python3
"

Reply:
/
/ #6112